### PR TITLE
Minor CLI improvement

### PR DESCRIPTION
**Pull request by dlon**
Follow-up to #5668. I realized that things could be made even slightly simpler by using `io::read_to_string` instead of `Read::read_to_string`.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676)

### CODE REVIEW

**Review by MarkusPettersson98 — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved

___
*[`mullvad-cli/src/cmds/patch.rs` line 9 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676#-NnrvXqCBe_gcdaWhq65:-NnrvXqCBe_gcdaWhq66:b-rby8sm) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/6eba7329c8602e73a86c07cd28293bc65b6884bd/mullvad-cli/src/cmds/patch.rs#L9)):*
> ```XML
> 
> /// If source is specified (not "-"), read from the provided file and send it as a settings patch to
> /// the daemon. Otherwise, read the patch from standard input.
> ```

Suggestion: I would rephrase this a little bit to be a bit more precise by distinguishing the two different cases. Something like:

```rust
/// Read a settings patch and send it to the daemon for validation and
/// application.
///
/// * If `source` is "-", read the patch from standard input
/// * Otherwise, interpret `source` as a filepath and read from the provided
/// file
```



_Code quote:_
```XML
/// If source is specified (not "-"), read from the provided file and send it as a settings patch to
/// the daemon. Otherwise, read the patch from standard input.
```
___
*[`mullvad-cli/src/cmds/patch.rs` line 30 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676#-NnrxE8J4wUQ1KD5Mkma:-NnrxE8J4wUQ1KD5Mkmb:bb7eybp) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/6eba7329c8602e73a86c07cd28293bc65b6884bd/mullvad-cli/src/cmds/patch.rs#L30)):*
> ```XML
> 
> /// If source is specified (not "-"), write a patch to the file. Otherwise, write the patch to
> /// standard output.
> ```

dito



_Code quote:_
```XML
/// If source is specified (not "-"), write a patch to the file. Otherwise, write the patch to
/// standard output.
```

**Review by dlon — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @MarkusPettersson98)

___
*[`mullvad-cli/src/cmds/patch.rs` line 9 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676#-NnrvXqCBe_gcdaWhq65:-NnsPiKEDfvrr2Y-D-lv:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/6eba7329c8602e73a86c07cd28293bc65b6884bd/mullvad-cli/src/cmds/patch.rs#L9)):*
<details><summary><i>Previously, MarkusPettersson98 (Markus Pettersson) wrote…</i></summary><blockquote>

Suggestion: I would rephrase this a little bit to be a bit more precise by distinguishing the two different cases. Something like:

```rust
/// Read a settings patch and send it to the daemon for validation and
/// application.
///
/// * If `source` is "-", read the patch from standard input
/// * Otherwise, interpret `source` as a filepath and read from the provided
/// file
```
</blockquote></details>

Done.
___
*[`mullvad-cli/src/cmds/patch.rs` line 30 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676#-NnrxE8J4wUQ1KD5Mkma:-NnsPhHYCDyrEIbVtP-k:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/6eba7329c8602e73a86c07cd28293bc65b6884bd/mullvad-cli/src/cmds/patch.rs#L30)):*
<details><summary><i>Previously, MarkusPettersson98 (Markus Pettersson) wrote…</i></summary><blockquote>

dito
</blockquote></details>

Done.

**Review by MarkusPettersson98 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5676#-:-NnsY37uAwEh-P-t-v-6:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved
